components/ActivityBarChart.tsx

```
@@ -0,0 +1,162 @@
+const BarItem: React.FC<BarItemProps> = ({
+    day,
+    value,
+    maxValue,
+    gradient,
+    delay,
+    bgColor,
+}) => {
+    const height = useSharedValue(0);
+    const { colors } = useTheme();
+
+    useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on delay.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)